packages/color/color.0.1.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "1.2"
+name: "color"
+maintainer: "anuragsoni.13@gmail.com"
+authors: ["Anurag Soni"]
+homepage: "https://github.com/anuragsoni/color"
+dev-repo: "git+https://github.com/anuragsoni/color.git"
+bug-reports: "https://github.com/anuragsoni/color/issues"
+doc: "https://anuragsoni.github.io/color/"
+license: "MIT"
+
+build: [
+  ["jbuilder" "subst"] {pinned}
```

Comment:
That needs to be changed (i.e. add a `"-p" name`). See  #12279

Reply:
@kit-ty-kate Thanks for pointing that out. I've updated the PR